docs/Antora/modules/howto_guides/pages/howto_measure_single_value.adoc

```diff
@@ -0,0 +1,25 @@
+= Measure a single value
+
+[#measure_single_value]
+== Measure a single value
+
+Sometimes you just want to get a single measurement value of a signal and do not need a streaming session.
```

Comment:
You can replace text with something like 

OpenDAQ offers a variety of solutions for reading signal values, including readers. For developers requiring access to only the most recent value, the signal interface provides a method named "getLastValue."

Reply:
Description updated